python/cuml/manifold/umap.pyx

```diff
@@ -348,7 +354,9 @@ class UMAP(UniversalBase,
                  callback=None,
                  handle=None,
                  verbose=False,
-                 output_type=None):
+                 output_type=None,
```

Comment:
I think that we usually leave the `output_type` parameter as the last one on most estimators.